lib/static/js/code-blocks-buttons.js

```diff
@@ -0,0 +1,47 @@
+/* global ClipboardJS */
+
+window.addEventListener('load', function() {
+  function button(label, ariaLabel, icon, className) {
+    const btn = document.createElement('button');
+    btn.classList.add('btn-icon', className);
```

Comment:
The classes within this file has to be updated for the renamed selectors.

Reply:
ugh, sorry that last commit was done so sloppily, fixed